diff --git a/node_modules/vcards-js/index.js b/node_modules/vcards-js/index.js
index 032aece..5ae2f7e 100644
--- a/node_modules/vcards-js/index.js
+++ b/node_modules/vcards-js/index.js
@@ -2,57 +2,11 @@
     vCards-js, Eric J Nesser, November 2014
 ********************************************************************************/
 /*jslint node: true */
-'use strict';
 
 /**
  * Represents a contact that can be imported into Outlook, iOS, Mac OS, Android devices, and more
  */
 var vCard = (function () {
-    /**
-     * Get photo object for storing photos in vCards
-     */
-    function getPhoto() {
-        return {
-            url: '',
-            mediaType: '',
-            base64: false,
-
-            /**
-             * Attach a photo from a URL
-             * @param  {string} url       URL where photo can be found
-             * @param  {string} mediaType Media type of photo (JPEG, PNG, GIF)
-             */
-            attachFromUrl: function(url, mediaType) {
-                this.url = url;
-                this.mediaType = mediaType;
-                this.base64 = false;
-            },
-
-            /**
-             * Embed a photo from a file using base-64 encoding (not implemented yet)
-             * @param  {string} filename
-             */
-            embedFromFile: function(fileLocation) {
-              var fs   = require('fs');
-              var path = require('path');
-              this.mediaType = path.extname(fileLocation).toUpperCase().replace(/\./g, "");
-              var imgData = fs.readFileSync(fileLocation);
-              this.url = imgData.toString('base64');
-              this.base64 = true;
-            },
-
-            /**
-             * Embed a photo from a base-64 string
-             * @param  {string} base64String
-             */
-            embedFromString: function(base64String, mediaType) {
-              this.mediaType = mediaType;
-              this.url = base64String;
-              this.base64 = true;
-            }
-        };
-    }
-
     /**
      * Get a mailing address to attach to a vCard.
      */
@@ -192,11 +146,7 @@ var vCard = (function () {
          */
         lastName: '',
 
-        /**
-         * Logo
-         * @type {object}
-         */
-        logo: getPhoto(),
+ 
 
         /**
          * Middle name
@@ -234,11 +184,7 @@ var vCard = (function () {
          */
         organization: '',
 
-        /**
-         * Individual's photo
-         * @type {object}
-         */
-        photo: getPhoto(),
+
 
         /**
          * The role, occupation, or business category of the vCard object within an organization
@@ -320,18 +266,6 @@ var vCard = (function () {
             var vCardFormatter = require('./lib/vCardFormatter');
             return vCardFormatter.getFormattedString(this);
         },
-
-        /**
-         * Save formatted vCard to file
-         * @param  {String} filename
-         */
-        saveToFile: function(filename) {
-            var vCardFormatter = require('./lib/vCardFormatter');
-            var contents = vCardFormatter.getFormattedString(this);
-
-            var fs = require('fs');
-            fs.writeFileSync(filename, contents, { encoding: 'utf8' });
-        }
     };
 });
 
